js/main.js

```diff
@@ -57,8 +57,8 @@ function hideMenu() {
 
 // Play/Pause Button
 function playPause() {
-  // Set media player variable.
-  var video = $('#bgvid')[0];
+  // Set media player constant.
+  const video = document.getElementById("bgvid");

```

Comment:
I understand the const part, but why change jquery to DOM here and leave all the rest (except few cases) jquery?